glossary.md

```diff
@@ -6,7 +6,7 @@ A [directory structure](bundle.md) that is written ahead of time, distributed, a
 
 ## Configuration
 
-The [`config.json`](config.md) and [`runtime.json`](runtime-config.md) files in a [bundle](#bundle) which define the intended [container](#container) and container process.
+The [`config.json`](config.md) file in a [bundle](#bundle) which define the intended [container](#container) and container process.
```

Comment:
On Mon, Jan 25, 2016 at 11:54:24AM -0800, Mrunal Patel wrote:
> > -The [`config.json`](config.md) and [`runtime.json`](runtime-config.md) files in a [bundle](#bundle) which define the intended [container](#container) and container process.
> > +The [`config.json`](config.md) file in a [bundle](#bundle) which define the intended [container](#container) and container process.
> 
> s/define/defines/

Thanks, fixed in 0d4a59b → 7610129.